src/content/docs/en/reference/error-reference.mdx

```diff
@@ -127,7 +127,6 @@ The following reference is a complete list of the errors you may encounter while
 ## Action Errors
 
 - [**ActionsWithoutServerOutputError**](/en/reference/errors/actions-without-server-output-error/)<br/>Actions must be used with server output.
-- [**ActionsUsedWithForGetError**](/en/reference/errors/actions-used-with-for-get-error/)<br/>An invalid Action query string was passed by a form.
 - [**ActionsReturnedInvalidDataError**](/en/reference/errors/actions-returned-invalid-data-error/)<br/>Action handler returned invalid data.
-- [**ActionQueryStringInvalidError**](/en/reference/errors/action-query-string-invalid-error/)<br/>An invalid Action query string was passed by a form.
+- [**ActionNotFoundError**](/en/reference/errors/action-not-found-error/)<br/>Action not found.
```

Comment:
Similarly here, it created a file for the new name but didn't remove the file for the old name